pkg/ingester/active_series.go

```diff
@@ -53,8 +53,8 @@ func NewActiveSeries() *ActiveSeries {
 }
 
 // Updates series timestamp to 'now'. Function is called to make a copy of labels if entry doesn't exist yet.
-func (c *ActiveSeries) UpdateSeries(series labels.Labels, now time.Time, labelsCopy func(labels.Labels) labels.Labels) {
-	fp := fingerprint(series)
```

Comment:
If we are not going to use `fingerprint(series)`, then it is not used anywhere else. Let's remove related code

Reply:
yes, has been removed.